RecoMET/METPUSubtraction/plugins/DeepMETProducer.cc

```diff
@@ -69,7 +69,7 @@ DeepMETProducer::DeepMETProducer(const edm::ParameterSet& cfg, const DeepMETCach
 void DeepMETProducer::produce(edm::Event& event, const edm::EventSetup& setup) {
   auto const& pfs = event.get(pf_token_);
 
-  static const tensorflow::NamedTensorList input_list = {
+  const tensorflow::NamedTensorList input_list = {
```

Comment:
Was the timing of this change checked? The reason is it is probably more efficient to make this a member data of the class so that it doesn't have to be recreated on each event call. That only matters if this constructor is relatively slow.

Reply:
a copy constructor of a tensor apparently only makes a copy of TensorShape (uint8[16] and int64) and of a pointer to TensorBuffer. 
So, it apparently should not cost much

Reply:
looking at either self-cost or identified calls in
https://slava77sk.web.cern.ch/slava77sk/reco/cgi-bin/igprof-navigator/CMSSW_11_2_X_2020-10-10-1100-test31738.test-NANO-102Xv1.100.pp/439
I do not see any evidence for non-negligible CPU cost